### ISSUE TITLE
iOS app only listen once.

### ISSUE THREAD

**Issue by iampawan** [alan-ai/alan-sdk-flutter#2]
There is an issue on iOS side, it only listens once and then can't. Android works fine

**Comment by snyuryev** [comment]
@iampawan please provide iOS version, flutter plugin version and steps how we can reproduce the issue.
What does mean "it only listens once"? 
What is going on after that?

**Comment by iampawan** [comment]
When you register it for the first time it works or alan activates but then it doesn't work. You have to restart your app again. I am using v2.0.11 with iOS 14.2.1. Let me know if you need any more info @snyuryev

**Comment by snyuryev** [comment]
@iampawan what is happening then you try to tap Alan Button second time?
That will be great if you provide some code to reproduce your issue.
Thanks.

**Comment by iampawan** [comment]
It doesn't respond after tapping again. Ok, I'll try to send you the code. But I guess it is not working even for a hello world app @snyuryev

**Comment by snyuryev** [comment]
@iampawan have you tried our flutter examples? Like Shire App?
https://github.com/alan-ai/alan-sdk-flutter/tree/master/examples/ShrineApp

**Comment by iampawan** [comment]
No, I haven't tried to run that. But the same thing is working fine on Android btw

**Comment by snyuryev** [comment]
@iampawan will wait for code from you.

Btw this is a good place to start - step-by-step tutorial for Flutter https://www.alan.app/docs/client-api/cross-platform/flutter

**Comment by iampawan** [comment]
Hey @snyuryev New finding - it is working fine on the simulator. Looks like a device issue on a particular version. I am using iPhone 12 with 14.2.1

**Comment by iampawan** [comment]
@snyuryev I have done the implementation a couple of times before. This issue came recently I guess

**Comment by snyuryev** [comment]
@iampawan did you run flutter app on different iOS device or iOS version previously?

**Comment by iampawan** [comment]
Yes but it’s few months back

**Comment by saimazansari** [comment]
Alan AI only listens not responds to the activity
